src/enhanced/types/job-types.ts

```diff
@@ -0,0 +1,26 @@
+export type JobType = 'githubPush' | 'manifestGeneration' | 'productionDeploy' | 'regression';
```

Comment:
NOTE: These properties will probably change in the near future, but I am keeping this for now as this is what we can potentially expect from the queue. This was added before I realized that we need to make changes with how we interact with the MongoDB collection.